translation_v2.json

```diff
@@ -13,7 +13,8 @@
             "antigen_strings.xml",
             "vaccination_strings.xml",
             "green_certificate_strings.xml",
-            "green_certificate_attribute_strings.xml"
+            "green_certificate_attribute_strings.xml",
+            "family_certificate_strings.xml"
```

Comment:
Please align with https://github.com/corona-warn-app/cwa-app-android/pull/3447

Reply:
@axelherbstreith  will unify the strings in one file , I will keep it here for now

Reply:
To avoid conflicts , I renamed my file into `covid_certificate_strings.xml` and deleted empty `recovery_certificate_strings.xml`